<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListWidget" name="entryListWidget"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="addEntryButton">
        <property name="text">
         <string>Add entry</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeEntryButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Remove entry (Not implemented)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editEntryButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Edit entry (Not implemented)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Save all entries</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="captureButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Capture gestures</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="dataVisualizer">
        <property name="undoRedoEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="tabStopWidth">
         <number>20</number>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>524</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
